opea-comps/mega-service/app.py

```
@@ -59,64 +85,175 @@ def start(self):
+        initial_inputs={
+            "messages": chat_request.messages,
+        }
+        print("\n\n\n\nPAYLOAD:\n")
+        print(json.dumps(initial_inputs))
+        print("\n\n\n\n")
+        result_dict, runtime_graph = await self.megaservice.schedule(
+            initial_inputs=initial_inputs,
+            llm_parameters=parameters
+        )
+        print("\n\nresult_dict:\n",result_dict)
+        for node, response in result_dict.items():
```

Ruff (0.8.2)

Loop control variable `node` not used within loop body

Rename unused `node` to `_node`

(B007)